docs/guide/javascript_and_css.md

```diff
@@ -8,6 +8,199 @@ parent: How-to guide
 
 While ViewComponent doesn't provide any built-in tooling to do so, it’s possible to include JavaScript and CSS alongside components.
 
+## Propshaft / Stimulus
+
+To use a [transpiler-less and bundler-less approach to JavaScript](https://world.hey.com/dhh/modern-web-apps-without-javascript-bundling-or-transpiling-a20f2755) (the default for Rails 8), Stimulus and CSS can be used inside ViewComponents one of two ways:
+
+### Upgrading a pre-Rails 8 app
```

Comment:
Let's remove this section since upgrading your app is outside ViewComponent's scope.

Reply:
You're right. I wish there had been a more straightforward guide for this elsewhere but makes sense, doesn't belong here =]